### DISCUSSION TITLE
Requests for species to add in the Nocmig model  (Europe)

### DISCUSSION THREAD

**Discussion by Mattk70** [Mattk70/Chirpity-Electron#144] [Ideas]
I'd like to hear about the European species that are missing from the Nocmig model (rather than BirdNET). I'm keen to broaden the range of Nocmig species' detections, within the European area.

So far, I've had requests to add the following:
* River Warbler
* Thrush Nightingale
* Tengmalms Owl
* Hazel Grouse 
* Great Reed Warbler
* Woodchat Shrike
* Tawny Pipit

Are there other species you'd like the Nocmig model to detect?

**Comment by tphakala** [comment]
Do you need audio samples for any of these species?

**Reply by Mattk70** [reply]
Hi @tphakala , thanks for the offer. It is quite likely to be useful yes, because as a rule more data is better! I would check the XC website for what they have to download first, No hurry though, I think it'll be a while before I am ready to start training a new model!

**Comment by paulcoiffard** [comment]
Hi Matt, I have extracted from Trektellen the species most recorded in France in nocmig and I have removed the species already present in Chirpity's ‘Nocturnal Birds’ species list. I kept only the species recorded in flight (I have removed the local species that are not in flight) but there are local species in flight in the list and probably also errors, particularly in the species recorded few times. This is just informative but it'll give you a good start ;) 

[Missing_species.txt](https://github.com/user-attachments/files/17586698/Missing_species.txt)

**Comment by Mattk70** [comment]
Thanks @paulcoiffard , however, the nocturnal birds list is not the full list of species known to the Nocmig model. Most of the species in your file are known to it. Would you mind finding the missing species by comparing the species included in  the "everything" list with what you found on Trektellen?

I have attached the full list for you:
[species_list.txt](https://github.com/user-attachments/files/17588076/species_list.txt)

**Reply by paulcoiffard** [reply]
Oops, that's make sense. I re-run my script with the full list and here's the result :
[Missing_species_everything.txt](https://github.com/user-attachments/files/17588466/Missing_species_everything.txt)

**Reply by Mattk70** [reply]
Thanks, that's a good list! There are a couple that do exist but show as missing because there are different taxonomies, clearly. Jackdaw &  Dotterel are already in the model. Feral pigeon is a Rock Dove in the model. I'm pretty sure they can't be separated on sound.
